official/resnet/resnet.py

```diff
@@ -608,3 +710,9 @@ def __init__(self, resnet_size_choices=None):
              'is not always compatible with CPU. If left unspecified, '
              'the data format will be chosen automatically based on '
              'whether TensorFlow was built for CPU or GPU.')
+
+    self.add_argument(
+        '-v', '--version', type=int, choices=[1, 2], dest="version",
+        metavar="", default=_DEFAULT_VERSION,
```

Comment:
What is metavar? Why have it if blank?

Reply:
metavar is how argparse displays the variable. For whatever reason, when you have multiple spellings of a flag with choices set, rather than display it in -h as:

`-v, --version {1, 2}`

it is displayed as:
`-v {1, 2}, --version {1, 2}`

which is ugly. Setting metavar="" suppresses that behavior.